cli.py

```diff
@@ -167,6 +190,10 @@ class Meta:
         arguments = [
             (['name'],
              dict(help='The item name.', action='store')),
+            (['-a', '--account'],
```

Comment:
I think that both `--account` and `--vault` should be a parameter for the cli itself, not for its subcommands (`item` and `search`).
If you agree and we can do that easily please make the change... If not, we can leave it like this to get this out of the door asap.